arches/app/templates/views/components/widgets/number.htm

```diff
@@ -5,7 +5,7 @@
 {% block form %}
 <div class="row widget-wrapper">
     <div class="form-group">
-        <label class="control-label widget-input-label" for="" data-bind="text:label"></label>
+        <label class="control-label widget-input-label" data-bind="text:label"></label>
```

Comment:
I think we should change the label element to a span element.  That way we don't have a label that doesn't point to an input element (which is flagged in the WAVE plugin). 
With that change we should also add this `display: inline-block;` to the `widget-input-label` class.